### PR TITLE
Add Bryson69

### LINKED ISSUE #423: Bryson Mundia
Bryson Mundia

### PR DESCRIPTION
**Pull request by Bryson69**
resolve #423

### CODE REVIEW

**Comment by Bryson69** [comment]
How does it look?

**Comment by Bryson69** [comment]
How does it look?

File: data.json [hunk 1]
```diff
@@ -712,5 +712,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/20727684/112530125-fb464380-8d9d-11eb-963a-0a8b25a7a87f.png",
     "issueNumber": 412
-  }
+  },
+  {
+    "name": "Bryson Mundia",
+    "githubUsername": "Bryson69",
+    "imageUrl": "https://user-images.githubusercontent.com/624760/91057573-48531300-e61f-11ea-9e13-2d7384e42000.png",
+    "issueNumber": 423
+}
```
Comment (AllanRegush) [inline data.json:721]: Please line up this bracket like the rest. Just for consistent style :+1:

**Review by AllanRegush — CHANGES_REQUESTED** [review]
Looking Good :+1: Just a minor adjustment. :nerd_face:

**Review by Bryson69 — COMMENTED** [review]
How does it look

**Comment by Bryson69** [comment]
Made the minor adjustment

**Review by Bryson69 — COMMENTED** [review]
How is it?

**Comment by Bryson69** [comment]
I have made the changes

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

**Pull request merged**